static/app/components/quickTrace/quickTrace.spec.tsx

```diff
@@ -123,12 +121,13 @@ describe('Quick Trace', function () {
           organization={organization}
         />
       );
-      expect(quickTrace.text()).toEqual('\u2014');
+      expect(container).toHaveTextContent('\u2014');
     });
 
-    it('renders partial trace with no children', function () {
+    // TODO
+    it('renders partial trace with no children', async function () {
       MockApiClient.warnOnMissingMocks();
-      const quickTrace = mountWithTheme(
+      const {findAllByTestId} = render(
```

Comment:
its recommended using `screen.findAllByTestId` instead of the functions returned by the render https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
done!